tensorflow_datasets/testing/test_data/fake_examples/en_tam_parallel_text/en-ta-parallel-v2/corpus.bcn.dev.en

```diff
@@ -0,0 +1,2 @@
+abacus
+abbreviated addressing
```

Comment:
like the others, you have to finish at the end of the line.

Reply:
Kokoro failed because of this error, it seems the 3 example on there.

```
E     AssertionError: Sizes do not match: 3 vs 2 for /tmpfs/src/github/datasets/tensorflow_datasets/testing/test_data/fake_examples/en_tam_parallel_text/en-ta-parallel-v2/corpus.bcn.dev.en vs /tmpfs/src/github/datasets/tensorflow_datasets/testing/test_data/fake_examples/en_tam_parallel_text/en-ta-parallel-v2/corpus.bcn.dev.ta.
tensorflow_datasets/translate/en_tam_parallel_text.py:232: AssertionError
```